pkg/systemd/quadlet/quadlet.go

```diff
@@ -672,6 +674,12 @@ func ConvertContainer(container *parser.UnitFile, names map[string]string, isUse
 		return nil, err
 	}
 
+	groupsAdd, ok := container.LookupAll(ContainerGroup, KeyGroupAdd)
```

Comment:
Sorry, missed this change
```suggestion
	groupsAdd := container.LookupAll(ContainerGroup, KeyGroupAdd)
```